### PR TITLE
Step 5 - New line breaks and commas should be interchangeable between numbers

### PR DESCRIPTION
**Pull request by preetinadkarni**
### Step 5 

New line breaks and commas should be interchangeable between numbers.  

~~~
Add("1,2\n3") > Returns 6
Add("3\n5\n3,9") > Returns 20
~~~

### CODE REVIEW

File: UnitTests/UnitTests.cs [hunk 1]
```diff
@@ -29,14 +29,16 @@ public int GivenInputIsASingleNumberWhenAddThenReturnsThatNumber(string input)
         
         [TestCase("1,2", ExpectedResult=3)]
         [TestCase("3,5", ExpectedResult=8)]
-        public int GivenInputIsTwoNumbersWhenAddThenReturnsSumOfNumbers(string input)
+        [TestCase("1,2,3", ExpectedResult=6)]
+        [TestCase("3,5,3,9", ExpectedResult=20)]
+        public int GivenInputIsAmountOfNumbersWhenAddThenReturnsSumOfNumbers(string input)
```
Comment (ddarylmae) [inline UnitTests/UnitTests.cs:34]: Hmmm, basing it off of your other test description, think a better name for this test would be something like _given separated by commas_

File: UnitTests/UnitTests.cs [hunk 2]
```diff
@@ -29,14 +29,16 @@ public int GivenInputIsASingleNumberWhenAddThenReturnsThatNumber(string input)
...
         {
             return _stringCalculator.Add(input);
         }
         
-        [TestCase("1,2,3", ExpectedResult=6)]
-        [TestCase("3,5,3,9", ExpectedResult=20)]
-        public int GivenInputIsAmountOfNumbersWhenAddThenReturnsSumOfNumbers(string input)
+        [TestCase("1,2\n3", ExpectedResult=6)]
+        [TestCase("3\n5\n3,9", ExpectedResult=20)]
+        public int GivenInputHasLineBreaksOrCommashenAddThenReturnsSumOfNumbers(string input)
```
Comment (ddarylmae) [inline UnitTests/UnitTests.cs:41]: `WhenAdd`?

**Review by ddarylmae — CHANGES_REQUESTED** [review]
Good progress! Just need minor changes on naming the tests

File: UnitTests/UnitTests.cs (hunk 2 above)
Comment (preetinadkarni) [reply UnitTests/UnitTests.cs:41]: good catch!

**Pull request merged**